packages/transport-bridge/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "@trezor/transport-bridge",
+    "version": "1.0.0",
```

Comment:
Total nit, which I am sure is too soon for, but it'd be great to version the node version of bridge as v3. That means 3.0.0. for users.